Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
I know, but it looks like an useful thing. Do you prefer removing it, or use this method to show a Headline only once?